riotfile.py

```diff
@@ -79,6 +79,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         # "pytest-cov": latest,
         "opentracing": latest,
         "hypothesis": latest,
+        "pep562": latest,
```

Comment:
is this needed here? if it is part of the packages dependencies then they should be getting installed?